CHANGES.md

```diff
@@ -64,6 +67,11 @@ To be released.
 
 ### Added interfaces
 
+ -  Users became able to give a cancellation token to `Block<T>.Mine()` and
+    `Hashcash.Answer()` to cancel the operation.  [[#460], [#517]]
```

Comment:
Although the added parameter has a default value, this makes a bytecode-level API (i.e., ABI) change so that library users need to recompile their code.  This should go to backward-incompatible interface changes.